scripts/generate_genotype_df/generate_genotype_df.py

```diff
@@ -13,9 +14,9 @@ def query():
     hl.init(default_reference='GRCh38')
 
     mt = hl.read_matrix_table(TOB_WGS)
+    mt = hl.experimental.densify(mt)
```

Comment:
Do you need this? Won't this just affect the reference blocks?

Reply:
I thought densifying was necessary whenever you use the `GT` field?

Reply:
Sorry, of course you're right. I keep forgetting about the actual semantics of `densify`. Loic put it pretty succinctly a while ago: "densify does not add or remove rows, ever. It simply fills up missing entries." (https://nbviewer.org/github/populationgenomics/tob-deepdive/blob/main/densify.ipynb).